build_cmake/configs/base.cmake

```diff
@@ -180,7 +192,20 @@ config_string(
 if(HAVE_DIAGNOSTIC AND (NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Debug"))
     # Avoid setting diagnostic flags if we are building with Debug mode.
     # CMakes Debug config sets compilation with debug symbols by default.
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g")
+    if("${CMAKE_C_COMPILER_ID}" STREQUAL "MSVC")
+        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /Z7")
```

Comment:
Could you add a comment explaining what '/Z7' is?

Reply:
Updated :)